src/data.js

```diff
@@ -176,9 +176,11 @@ c3_chart_internal_fn.mapTargetsToUniqueXs = function (targets) {
 };
 c3_chart_internal_fn.addHiddenTargetIds = function (targetIds) {
     this.hiddenTargetIds = this.hiddenTargetIds.concat(targetIds);
```

Comment:
shall I wait #1663 to be merged. Because I would place my line in the if statement. I also dont want to increase or decrease `this.hiddenTargetCount` if not every ID in `targetIds` is added to `this.hiddenTargetIds`